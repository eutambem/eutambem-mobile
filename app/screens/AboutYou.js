import React from 'react';
import PropTypes from 'prop-types';

import Container from '../components/common/Container';
import AboutYouForm from '../components/AboutYouForm';

const AboutYou = props => (
  <Container>
    <AboutYouForm navigation={props.navigation} />
  </Container>
);

AboutYou.navigationOptions = {
  headerTitle: 'Sobre vocÃª',
};

AboutYou.propTypes = {
  navigation: PropTypes.shape({
    navigate: PropTypes.func,
  }).isRequired,
};

export default AboutYou;
