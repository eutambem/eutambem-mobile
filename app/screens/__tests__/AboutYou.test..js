import 'react-native';
import React from 'react';
import { shallow } from 'enzyme';
import sinon from 'sinon';

import AboutYou from '../AboutYou';

const props = {
  navigation: {
    navigate: sinon.spy(),
  },
};

describe('AboutYou', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<AboutYou {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
