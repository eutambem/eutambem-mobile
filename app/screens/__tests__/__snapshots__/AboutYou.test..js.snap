// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutYou renders correctly 1`] = `
<Container
  innerContainerStyle={Object {}}
>
  <AboutYouForm
    navigation={
      Object {
        "navigate": [Function],
      }
    }
  />
</Container>
`;
